docs/source/router.rst

```diff
@@ -0,0 +1,97 @@
+Router
+======
+
+Working with Routers
+````````````````````
+
+Real world applications can almost never fit all logic into a single file.
+As Flask supports splitting your API into smaller modules via Blueprints, in FlaskNinja the module is called Router.
+
+When you create a new Router instance, you can register your endpoints there as to the Ninja API instance:
```

Comment:
I am not sure if I understand this sentence, can you please explain?

Reply:
I rewrote the text, so it is more understandable